django_extensions/management/commands/sqldiff.py

```diff
@@ -354,7 +354,7 @@ def get_field_db_type(self, description, field=None, table_name=None):
         tablespace = field.db_tablespace
         if not tablespace:
             tablespace = "public"
-        if (tablespace, table_name, field.column) in self.unsigned:
+        if (tablespace, table_name, field.column) in self.unsigned and self.unsigned_suffix not in field_db_type:
```

Comment:
Have not tested it but should we use `... and (self.unsigned_suffix is not None and self.unsigned_suffix not in field_db_type):` here ?

To avoid `TypeError: 'in <string>' requires string as left operand, not NoneType`

Reply:
`unsigned_suffix` is a class variable and essentially a constant so it should be safe, but if you'd rather have belt and braces that's fine too.

Reply:
I was looking that it's only set in `MySQLDiff` and otherwise set to `None` resulting in `TypeError`.
However if it's set to `None` then the `self.unsigned` set would also be empty :-)